spec/controllers/followups_controller_spec.rb

```diff
@@ -97,7 +97,7 @@
         get :new, params: { :request_id => request.id,
                             :incoming_message_id => message_id }
         expect(response.body).
-          to have_css("div#other_recipients ul li", :text => "the main FOI contact address for")
+          to have_css("div#followup", :text => "the main FOI contact at")
```

Comment:
Not sure about this change as the assertion has now changed its meaning.

Originally we were checking that _if_ there are alternative reply addresses available, then we should _also_ list the main contact address for selection.

Now its just asserting that somewhere we say "you're replying to the main contact address".

The change here should be in the spec setup, with the original assertion.